app/frontend/types.ts

```diff
@@ -56,11 +65,6 @@ export type _PubKeyCbor = {
   cborHex: HexString
 }
 
-export type _Address = {
```

Comment:
seems to me like "DerivedAddress" could be a good name for this type as it would convey the information that we derived the address and therefore it's natural to expect some additional context like the derivation path and possibly encoding as well